### PR TITLE
Drop python 2 support

### PR DESCRIPTION
**Pull request by xrmx**
It's probably time to let python2 support go.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -4,14 +4,12 @@
 
 [tox]
 envlist =
-        py27-django{1.11},
-        py36-django{1.11,2.2,3.0}
+        py36-django{2.2,3.0}
```
Comment (DmytroLitvinov) [inline tox.ini]: Also would be nice to see here Python3.7

Comment (xrmx) [reply tox.ini]: There's a plenty more python versions we can test against but given the scope of this project I don't see much value

**Comment by xrmx** [comment]
Update PR to actually run tests (on github actions) and then make the tests pass with django 4.0.

**Comment by xrmx** [comment]
@imdario any chance you can look at this? If you don't have time I can help you with the maintenance of the project.

**Review by darccio — APPROVED** [review]

**Comment by darccio** [comment]
@xrmx Cool, I didn't see the PR. I'm merging it and giving you access for maintenance. Thank you very much!

**Comment by xrmx** [comment]
@imdario thanks!

**Comment by DmytroLitvinov** [comment]
Hi @imdario , and what about releasing it at PyPI? Could you give also access to it to @xrmx.
I can help also with maintaining it and releasing it.

**Comment by darccio** [comment]
@DmytroLitvinov Sure. I had to do some errands, but I plan to release it.

**Pull request merged**